site/src/components/Footer.tsx

```diff
@@ -119,7 +126,7 @@ const Socials = (
               outlineColor: (theme) => theme.palette.gray[50],
             },
             svg: {
-              padding: 0.3,
+              fontSize: 20,
```

Comment:
Just want to check this is deliberate: this reduces the size of the icons. If you haven't done so, perhaps worth checking w/ @MaggieAppleton what the

Previously:
<img width="379" alt="image" src="https://user-images.githubusercontent.com/42802102/157707652-7719fe93-f1a1-49cc-b077-5955cf1e2005.png">
Now:
<img width="369" alt="image" src="https://user-images.githubusercontent.com/42802102/157707679-07bf2988-80b1-4565-8b79-59e566998f54.png">

Reply:
I'm fine w/ this one - actually think the new looks better.